packages/expo-cli/src/commands/init.ts

```diff
@@ -472,5 +523,7 @@ export default function(program: Command) {
     .option('--yarn', 'Use Yarn to install dependencies. (default when Yarn is installed)')
     .option('--name [name]', 'The name of your app visible on the home screen.')
     .option('--yes', 'Use default options. Same as "expo init . --template blank')
+    .option('--android-package [name]', 'The package name for your Android standalone app')
+    .option('--ios-bundle-identifier [name]', 'The bundle identifier for your iOS standalone app')
```

Comment:
Maybe something like this 🤔 
```suggestion
    .option('--bundle-id [name]', 'The bundle identifier for your iOS standalone app')
```

Reply:
Yeah, it's a bit long for ios 😄 We could do `--package`/`--package-name` and `--bundle-id`?

Reply:
Didn't realize it was already a feature, we should probably keep it how it was https://github.com/expo/expo-cli/issues/824